verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -182,9 +210,41 @@ VName KytheFactsExtractor::ExtractVariableReference(
   }
 }
 
-VName KytheFactsExtractor::PrintAnchorVName(const Anchor& anchor,
-                                            absl::string_view file_path) {
-  const VName anchor_vname(file_path,
+VName KytheFactsExtractor::ExtractFunctionOrTask(
+    const IndexingFactNode& function_fact_node) {
+  const auto& function_name = function_fact_node.Value().Anchors()[0];
+
+  const VName function_vname(
+      file_path_, CreateScopeRelativeSignature(
+                      CreateFunctionOrTaskSignature(function_name.Value())));
+
+  const VName function_vname_anchor = PrintAnchorVName(function_name);
+
+  *stream_ << Fact(function_vname, kFactNodeKind, kNodeFunction);
+  *stream_ << Fact(function_vname, kFactComplete, kCompleteDefinition);
+  *stream_ << Edge(function_vname_anchor, kEdgeDefinesBinding, function_vname);
```

Comment:
Will these change to your new `Generate` functions once those are merged?

Reply:
Yes.